homeassistant/components/comfoair/sensor.py

```diff
@@ -0,0 +1,157 @@
+"""Platform to control a Zehnder ComfoAir 350 ventilation unit."""
+
+import logging
+
+from homeassistant.const import CONF_RESOURCES, TEMP_CELSIUS
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity import Entity
+from . import DOMAIN, ComfoAirModule, SIGNAL_COMFOAIR_UPDATE_RECEIVED
+from bitstring import BitArray
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_COMFORT_TEMPERATURE = "comfort_temperature"
+ATTR_OUTSIDE_TEMPERATURE = "outside_temperature"
+ATTR_SUPPLY_TEMPERATURE = "supply_temperature"
+ATTR_RETURN_TEMPERATURE = "return_temperature"
+ATTR_EXHAUST_TEMPERATURE = "exhaust_temperature"
+ATTR_AIR_FLOW_SUPPLY = "air_flow_supply"
+ATTR_AIR_FLOW_EXHAUST = "air_flow_exhaust"
+ATTR_FAN_SPEED_MODE = "speed_mode"
+
+SENSOR_TYPES = {
+    ATTR_COMFORT_TEMPERATURE: [
+        "Comfort Temperature",
+        TEMP_CELSIUS,
+        "mdi:thermometer",
+        0xD2,
```

Comment:
This is protocol specific information that doesn't belong in home assistant. Please abstract that into the device library.